app/components/page_list_component/view.html.erb

```diff
@@ -1,14 +1,50 @@
-<% @pages.each do |page| %>
-  <dl class="govuk-summary-list">
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key govuk-!-font-weight-regular">
-      <%= question_text_with_optional_suffix(page) %>
-      </dt>
-      <dd class="govuk-summary-list__actions">
-      <%= govuk_link_to edit_page_path(@form_id, page) do %>
-        <%= t("forms.form_overview.edit") %> <span class="govuk-visually-hidden"><%= page.question_text %></span>
-      <% end %>
-      </dd>
-    </div>
-  </dl>
+<%if FeatureService.enabled?(:reorder_pages) %>
+  <%= form_with url: move_page_url(@form_id), method: :post, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+    <% @pages.each_with_index do |page, index| %>
+      <dl class="govuk-summary-list govuk-!-margin-bottom-0">
+        <div class="govuk-summary-list__row">
+
+          <dt class="govuk-summary-list__key govuk-!-font-weight-regular">
+            <%= question_text_with_optional_suffix(page) %>
+          </dt>
+
+          <dd class="govuk-summary-list__actions govuk-!-padding-bottom-6">
+
+            <div class="govuk-button-group form-action-group govuk-button-group-right" }">
+              <% if show_up_button(index) %>
+                <%= f.govuk_submit t("forms.form_overview.move_up"), secondary: true, name: 'move_direction[up]', value: page.id %>
+                <span class="govuk-visually-hidden"><%= page.question_text %></span>
```

Comment:
Think the hidden text needs to be inside the button or it won't be part of the accessible name. You can probably do this by moving the `<span class="govuk-visually-hidden"></span>` into the translation and then passing the question name into it from this view

Reply:
Good spot and fix! 👍